opencraft/settings.py

```diff
@@ -295,9 +295,9 @@
 # for provisioning Open edX app servers.
 DEFAULT_CONFIGURATION_VERSION = env('DEFAULT_CONFIGURATION_VERSION', default=DEFAULT_OPENEDX_RELEASE)
 
-# Git tag for latest Open edX release. Used as a default refspec for
+# Git ref for stable Open edX release. Used as a default refspec for
 # configuration, edx-platform, forum, notifier, xqueue, and certs when creating production instances.
-LATEST_OPENEDX_RELEASE = env('LATEST_OPENEDX_RELEASE', default='named-release/dogwood.rc')
+OPENEDX_RELEASE_STABLE_REF = env('OPENEDX_RELEASE_STABLE_REF', default='named-release/dogwood.rc')
```

Comment:
I'm curious if you tried `named-release/dogwood.3` ? Because that tag should also be available on all these repos (if using the `edx` fork, but not `open-craft` fork etc). And I think that's nicer.

No need to change this though.